common/src/utils/math.ts

```diff
@@ -77,12 +77,12 @@ export const distance = (a: Vector, b: Vector): number => Math.sqrt(((b.x - a.x)
  */
 export const distanceSquared = (a: Vector, b: Vector): number => ((b.x - a.x) ** 2) + ((b.y - a.y) ** 2);
 
-export function lerp(start: number, end: number, percentage: number): number {
```

Comment:
Naming this percentage was technically wrong because a percentage usually ranges from 0 to 100, whereas this ranges from 0 to 1.